source/openapi-spec.yml

```diff
@@ -357,3 +400,16 @@ components:
           type: string
           description: A brief written description of the work involved
           example: "I lead, develop and enhance the literacy teaching practice of others..."
+    Error:
+      type: object
+      additionalProperties: false
+      properties:
+        code:
+          type: string
+          description: Http error code
```

Comment:
```suggestion
          description: HTTP error code
```

Reply:
From chatting with @duncanjbrown. 
We decided that `code` seems quite unnecessary as the code is return in the http request, so have been removed and replaced with `error` that gives a name for the given error.